src/components/PageContent/PageContent.astro

```diff
@@ -1,49 +1,204 @@
 ---
-import MoreMenu from "../RightSidebar/MoreMenu.astro"
-import TableOfContents from "../RightSidebar/TableOfContents"
-import WhatsNext from "./WhatsNext.astro"
-const { content, githubEditUrl, headings } = Astro.props
-const title = content.title
-const whatsNext = content.whatsnext
+import Overview from "./Overview.astro"
+import { MarkdownHeading } from "astro"
+
+export type Props = {
+  titleHeading: MarkdownHeading
+}
+const { titleHeading } = Astro.props
 ---
 
-<article id="article" class="content">
-  <section class="main-section">
-    <h1 class="content-title" id="overview">{title}</h1>
-    <nav class="block sm:hidden">
-      <TableOfContents client:media="(max-width: 50em)" headers={headings} />
-    </nav>
-    <slot />
-
-    <slot name="after-content" />
-    {whatsNext && <WhatsNext content={whatsNext} />}
-  </section>
-  <nav class="block sm:hidden">
-    <MoreMenu editHref={githubEditUrl} />
-  </nav>
+<article id="article">
+  <Overview title={titleHeading} />
+  <slot />
 </article>
 
-<style>
-  .content {
-    padding: 0;
-    /* max-width: 75ch; */
+<style is:inline>
```

Comment:
```suggestion
<style is:global>
```

Reply:
@fmonper1 Elaborate on this? I opt to have these styles `inline` rather than `global` since they're only meant to affect the HTML within this component. The content comes from `<slot>` so styles need to be at least `inline` to work properly